kubernetes/windows-exporter-daemonset.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    app: windows-node-exporter
+  name: windows-node-exporter
+  namespace: monitoring
+spec:
+  selector:
+    matchLabels:
+      app: windows-node-exporter
+  template:
+    metadata:
+      labels:
+        app: windows-node-exporter
+    spec:
+      securityContext:
+        windowsOptions:
+          hostProcess: true
+          runAsUserName: "NT AUTHORITY\\system"
+      hostNetwork: true
+      containers:
+      - args: 
+        - --config.file=%CONTAINER_SANDBOX_MOUNT_POINT%/config.yml
+        name: windows-node-exporter
+        image: jsturtevant/windows-exporter:latest
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 9182
+          hostPort: 9182
+          name: http
+        volumeMounts:
+        - name:  windows-exporter-config
+          mountPath: /config.yml
+          subPath: config.yml
+      nodeSelector:
+        kubernetes.io/os: windows
+      volumes:
+      - name: windows-exporter-config
+        configMap:
+          name: windows-exporter-config
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: windows-exporter-config
+  namespace: monitoring
+  labels:
+    app: windows-node-exporter
+data:
+  config.yml: |
+    collectors:
+      enabled: cpu,cs,net,logical_disk,os,system,service,container
+    collector:
+      service:
+        services-where: "Name='windows_exporter'"
```

Comment:
this service wont exist if running in a container?

Reply:
we could user containerd and kubelet as an example?

Reply:
good catch :-)